.gitattributes

```diff
@@ -55,3 +55,8 @@ phpunit.xml.dist export-ignore
 tests/test_app export-ignore
 tests/TestCase export-ignore
 .github export-ignore
+.mailmap export-ignore
+.varci.yml export-ignore
+phpcs.xml.dist export-ignore
+phpstan.neon export-ignore
```

Comment:
This needs to be kept. If someone wants to use phpstan for their app then can include this config file in their app's phpstan.neon.